.github/workflows/update-and-publish-helm-chart.yml

```diff
@@ -85,6 +94,11 @@ jobs:
             echo "New version ${NEW_CROMWELL_HELM_V} does not match GCP version ${NEXT_VERSION_GCP}"
             exit 1
           fi
+          
+          if [ "${NEW_CROMWELL_HELM_V}" != "${NEXT_VERSION_WDS}" ]; then
+            echo "New version ${NEW_CROMWELL_HELM_V} does not match WDS version ${NEXT_VERSION_WDS}"
```

Comment:
I know this is just the same as the line above, but what's the purpose here?  To keep all the charts in the same version even though they're separate?

Reply:
It seems we move in lockstep in regards to semantic versioning (see other examples under `charts/`)

In terms of why we do this....I'm not sure....haha